flowmachine/flowmachine/features/subscriber/active_subscribers.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+from datetime import timedelta, date, datetime
+from typing import List, Union, Optional
+from flowmachine.core.query import Query
+from flowmachine.features.subscriber.call_days import CallDays
+from flowmachine.features.subscriber.interevent_interval import IntereventInterval
+from flowmachine.features.utilities.events_tables_union import EventsTablesUnion
+
+
+class ActiveSubscribers(Query):
```

Comment:
I see the value of this in parallel with `UniqueSubscribers`, which gives you active subscribers for a period/day without the 'n hours each day' bit. Although it is also kind of duplicating the functionality of `TotalActivePeriods` + a `numeric_subset` (e.g. I think you could produce with a convenience function that makes a union of subset `TotalActivePeriods` with a tiny bit of SQL for window func).